dev_guide/builds.adoc

```diff
@@ -33,17 +33,22 @@ link:#docker-strategy-options[options])
 (link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[description],
 link:#custom-strategy-options[options])
 
-And there are three types of build source:
+And there are four types of build source:
 
 - link:#source-code[Git]
 - link:#dockerfile-source[Dockerfile]
+- link:#image-source[Image]
 - link:#binary-source[Binary]
 
 It is up to each build strategy to consider or ignore a certain type of source,
 as well as to determine how it is to be used.
 
 Binary and Git are mutually exclusive source types, while Dockerfile can be used
-by itself or together with Git and Binary.
+by itself or together with Git and Binary.  Image is intended to together with the
```

Comment:
ok, that is what i thought after looking at the code, but could have swore during our scrum discussion that someone declared it had to be used on conjunction with one of the others ... thx for the confirm